packages/algob/test/builtin-tasks/compile.ts

```diff
@@ -53,18 +53,22 @@ describe("Compile task", () => {
   const cacheDir = path.join("artifacts", "cache");
   const f1 = "asc-fee-check.copy.teal";
   const f2 = "asc-fee-check.teal";
+  const f3 = "gold-asa.py";
+  const f4 = "gold-asa.teal";
   const fhash = 2374470440; // murmur3 hash for f1 file
 
   it("on first run it should compile all .teal sources", async () => {
     await op.resetAndCompile(false);
 
-    assert.equal(op.timestamp, 2);
+    assert.equal(op.timestamp, 4);
```

Comment:
why the timestamp changed?

Reply:
because , instead of 2 files, now 2 more files are passed in `compile`
![check123](https://user-images.githubusercontent.com/26249903/97169643-61f41080-17b0-11eb-91c4-a87eb64fb50a.png)

Reply:
ok